app/repo/install.sh

```diff
@@ -0,0 +1,3 @@
+curl -s https://deb.siderus.io/key.asc | sudo apt-key add -
```

Comment:
Should we change the location of this file into (maybe) `/docs`?

Reply:
Location doesn't matter as long as it gets copied into the /repo so one can do `curl https://deb.siderus.io/install.sh | sudo bash -` for a one-line repo-add

Reply:
And don't forget to add the public key in there as well.